pennylane/devices/default_clifford.py

```diff
@@ -306,6 +328,35 @@ def circuit(state):
         >>> circuit(basis_states[2])
         tensor(0.0, requires_grad=True)
 
+    .. details::
+        :title: Error Channels
+
+        This device supports the ``finite-shot`` execution of the quantum circuits with
+        the following error channels that add ``Pauli noise``, allowing for one to perform
+        any sampling-based `measurements <https://github.com/quantumlib/stim/>`_.
```

Comment:
[optional] Can remove the last part:
"allowing for one to perform any sampling-based `measurements <https://github.com/quantumlib/stim/>`_."